.travis.yml

```diff
@@ -1,29 +1,35 @@
 language: ruby
+cache: bundler
+
 before_install:
   - gem install -v '1.17.3' bundler
-install: bundle _1.17.3_ install
-bundler_args: --jobs 3 --retry 3 --without benchmark
+bundler_args: --without benchmark
```

Comment:
I believe since `install` was overridden, `bundler_args` were not used. The default install action is `--jobs 3 --retry 3` and `bundler_args` will amend that. But let's see.

https://docs.travis-ci.com/user/languages/ruby/#dependency-management

Reply:
This option is actually outdated. I replaced it.